packages/rust/build.sh

```diff
@@ -65,6 +65,7 @@ termux_step_configure() {
 	mv $TERMUX_PREFIX/lib/libtinfo.so.6 $TERMUX_PREFIX/lib/libtinfo.so.6.tmp
 	mv $TERMUX_PREFIX/lib/libz.so.1 $TERMUX_PREFIX/lib/libz.so.1.tmp
 	mv $TERMUX_PREFIX/lib/libz.so $TERMUX_PREFIX/lib/libz.so.tmp
+	mv $TERMUX_PREFIX/lib/liblzma.so.$(. $TERMUX_SCRIPTDIR/packages/liblzma/build.sh; echo $TERMUX_PKG_VERSION) $TERMUX_PREFIX/lib/liblzma.so.tmp
```

Comment:
Can you extract this liblzma version as a variable once and just reuse that? I don't like seeing this extracted three times.